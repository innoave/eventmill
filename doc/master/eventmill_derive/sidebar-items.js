window.SIDEBAR_ITEMS = {"derive":[["AggregateType",""],["EventType",""]]};