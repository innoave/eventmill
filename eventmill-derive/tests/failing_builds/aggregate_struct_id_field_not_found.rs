use eventmill_derive::AggregateType;

#[derive(AggregateType, Debug)]
#[id_field(turtle_id)]
pub struct Turtle {
    id: String,
    x: f32,
    y: f32,
    direction: f32,
    speed: f32,
    pen: bool,
}

fn main() {}
