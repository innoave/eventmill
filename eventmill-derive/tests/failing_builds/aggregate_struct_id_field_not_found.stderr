error: no field turtle_id on struct Turtle
 --> $DIR/aggregate_struct_id_field_not_found.rs:4:12
  |
4 | #[id_field(turtle_id)]
  |            ^^^^^^^^^
